openpype/hosts/blender/plugins/create/create_render.py

```diff
@@ -0,0 +1,302 @@
+"""Create render."""
+import os
+
+import bpy
+
+from openpype.pipeline import (
+    get_current_project_name,
+)
+from openpype.settings import (
+    get_project_settings,
+)
+from openpype.pipeline import get_current_task_name
+from openpype.hosts.blender.api import plugin, lib
+from openpype.hosts.blender.api.pipeline import AVALON_INSTANCES
+
+
+class CreateRenderlayer(plugin.Creator):
+    """Single baked camera"""
+
+    name = "renderingMain"
+    label = "Render"
+    family = "renderlayer"
+    icon = "eye"
+
+    @staticmethod
+    def get_default_render_folder(settings):
+        """Get default render folder from blender settings."""
+
+        return (settings["blender"]
+                        ["RenderSettings"]
+                        ["default_render_image_folder"])
+
+    @staticmethod
+    def get_aov_separator(settings):
+        """Get aov separator from blender settings."""
+
+        aov_sep = (settings["blender"]
+                           ["RenderSettings"]
+                           ["aov_separator"])
+
+        if aov_sep == "dash":
+            return "-"
+        elif aov_sep == "underscore":
+            return "_"
+        elif aov_sep == "dot":
+            return "."
+        else:
+            raise ValueError(f"Invalid aov separator: {aov_sep}")
+
+    @staticmethod
+    def get_image_format(settings):
+        """Get image format from blender settings."""
+
+        return (settings["blender"]
+                        ["RenderSettings"]
+                        ["image_format"])
+
+    @staticmethod
+    def get_multilayer(settings):
+        """Get multilayer from blender settings."""
+
+        return (settings["blender"]
+                        ["RenderSettings"]
+                        ["multilayer_exr"])
+
+    @staticmethod
+    def get_render_product(output_path, name):
+        """
+        Generate the path to the render product. Blender interprets the `#`
+        as the frame number, when it renders.
+
+        Args:
+            file_path (str): The path to the blender scene.
+            render_folder (str): The render folder set in settings.
+            file_name (str): The name of the blender scene.
+            instance (pyblish.api.Instance): The instance to publish.
+            ext (str): The image format to render.
+        """
+        output_file = os.path.join(output_path, name)
+
+        render_product = f"{output_file}.####"
```

Comment:
This could be
```py
render_product = f"{os.path.join(output_path, name)}.####"
```

Reply:
Hound will complain about the second one for it being too long 😅 but the first one works, I'll change it :)